lib/plugins/plugin-priorities.js

```diff
@@ -10,20 +10,19 @@
 'use strict';
 
 module.exports = {
-    ExtractTextWebpackPlugin: 0,
+    MiniCssExtractPlugin: 0,
     DeleteUnusedEntriesJSPlugin: 0,
+    EntryFilesManifestPlugin: 0,
```

Comment:
should have a low priority, to be sure it runs after other plugins (btw, we should really change these priorities. Having them all at 0 makes the priority system a simple `prepend/append` one, which could then be simplified a lot)